src/analysis/retail/rogue/outlaw/modules/features/Checklist/Component.tsx

```diff
@@ -108,6 +108,26 @@ const OutlawRogueChecklist = ({
           thresholds={thresholds.opportunity}
         />
       </Rule>
+      <Rule
+        name="Make sure to use your audacity procs"
```

Comment:
```suggestion
        name=<>Make sure to use your <SpellLink id={TALENTS.AUDACITY_TALENT.id}/> procs</>
```

Also, change usages of SPELLS.AUDACITY to TALENTS.AUDACITY_TALENT